db/migrateWpPostsToArchieMl.ts

```diff
@@ -123,9 +123,9 @@ const migrate = async (): Promise<void> => {
                     refs: refParsingResults,
                 },
                 published: false, // post.published_at !== null,
-                createdAt: post.updated_at, // TODO: this is wrong but it doesn't seem that wordpress tracks the creation date
+                createdAt: post.updated_at_in_wordpress, // TODO: this is wrong but it doesn't seem that wordpress tracks the creation date
```

Comment:
you got some nice type errors here...

Reply:
@marcelgerber sorry am I blind? What is the typo? Or do you mean that "Updated at in wordpress" does not read well? If so do you have a better suggestion?

Reply:
Ah, sorry, I didn't realize that GH only shows the actual error on the Files tab:

> Type 'Date | null' is not assignable to type 'Date'.

Reply:
Ah right, my bad - thanks!